protocols/kad/src/behaviour.rs

```diff
@@ -2294,3 +2453,22 @@ impl fmt::Display for NoKnownPeers {
 }
 
 impl std::error::Error for NoKnownPeers {}
+
+/// The possible outcomes of [`Kademlia::add_address`].
+pub enum RoutingUpdate {
+    /// The given peer and address has been added to the routing
+    /// table.
```

Comment:
```suggestion
    /// table or was already part of the routing table before in which case any additional addresses were added to the entry.
```

Reply:
In what way do you think this is an improvement? This is the result of `add_address` which takes a peer and an address. Success means the given peer and address are now in the routing table, I don't think it matters whether they were already in there.